lab_1_classify_by_unigrams/main.py

```diff
@@ -12,6 +12,16 @@ def tokenize(text: str) -> list[str] | None:
     :return: a list of lower-cased tokens without punctuation
     """
 
+    if not isinstance(text, str):
+        return None
+    text = text.lower()
+    new_text = ""
+    for symbol in text:
+        if symbol.isalpha():
+            new_text += symbol
+    tokens = list(new_text)
```

Comment:
Consider creating a list right away instead of creating a string and only then converting it to list

Reply:
Solved